packages/ui/components/Socials.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="links">
+    <a class="link" :href="urls.github" target="_blank">
```

Comment:
Could we create an array with the socials and render the socials using an `v-for`?